core/templates/dev/head/css/oppia.css

```diff
@@ -60,6 +60,12 @@
   100% { opacity: 1; }
 }
 
+@-webkit-viewport{width:device-width}
```

Comment:
We could also just not add these rules, since I don't actually know of a concrete example where they are needed.